src/modules/ie-objects/elasticsearch/queryBuilder.ts

```diff
@@ -243,7 +260,18 @@ export class QueryBuilder {
 			},
 		];
 
-		if (user.isKeyUser && !isNil(user.sector)) {
+		if (isConsultableRemote || isConsultableMedia) {
+			checkSchemaLicenses = [
+				...checkSchemaLicenses,
+				{
+					terms: {
+						schema_license: [IeObjectLicense.BEZOEKERTOOL_CONTENT],
+					},
+				},
+			];
+		}
```

Comment:
**isConsultableRemote** => can i view this item's essence without visiting the visitor space.
**isOnlyConsultableOnLocation** => can i view this item's essence only by visiting the visitor space and never remotely

These 2 booleans are mutually exclusive, but it seems like you implemented **isAtLeastConsultableOnLocation** instead of either **isOnlyConsultableOnLocation** or **isConsultableRemote**

Also shouldn't **isConsultableMedia** take into account more than just the BEZOEKERTOOL_CONTENT license? It should filter for all objects where the essence is retuned, which also can happen through the VIAA-INTRA_CP-CONTENT license.

Reply:
> isOnlyConsultableOnLocation => can i view this item's essence only by visiting the visitor space and never remotely
I don't see a way to validate this one. I have an array of visitorSpaceIds but I can't validatie with a visitor space -> that only happens when the es query is executed.